frontend/src/container/TopNav/DateTimeSelectionV2/index.tsx

```diff
@@ -404,16 +405,25 @@ function DateTimeSelection({
 		time: Time,
 		currentRoute: string,
 	): Time | CustomTimeType => {
+		// if the relativeTime param is present in the url give top most preference to the same
+		if (relativeTimeFromUrl != null) {
+			return relativeTimeFromUrl as Time;
```

Comment:
Let's check how it will behave if we get corrupted relative time. It should fallback to the next priority value

Reply:
added validity checks here